samples/csharp/MicrosoftEduGraphSamples/workflows/Assignment.cs

```diff
@@ -0,0 +1,80 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.Extensions.Configuration;
+using Microsoft.Graph;
+using MicrosoftGraphSDK;
+
+namespace MicrosoftEduGraphSamples.Workflows
+{
+    /// <summary>
+    /// Contains all the workflows related to Assignment
+    /// </summary>
+    internal class Assignment
```

Comment:
Assignment-> AssignmentWorkflow

Reply:
We are following the pattern to group the samples by entity type (in this case Assignment) and all the workflows are located in this class.

Reply:
My reason of not naming class same as assignment entities is to avoid confusion for the developers